settings/Civisocial.setting.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+return array(
+  'enable_facebook' => array(
+    'group_name' => 'CiviSocial Account Credentials',
```

Comment:
@pradpnayak I don't see any suitable group there. Should we add a new group there?